ansible/roles/nbp-installer/tasks/main.yml

```diff
@@ -19,6 +19,7 @@
     state: present
   with_items:
     - open-iscsi
+  when: nbp_plugin_type != "hotpot_only"
```

Comment:
I thought this package is needed for hotpot as well, i.e., host-based replication.

Reply:
Got it